bin/ert-runner

```diff
@@ -2,11 +2,16 @@
 
 ERT_RUNNER="$(dirname $(dirname $0))/ert-runner.el"
 
-if [[ -n "$INSIDE_EMACS" ]]; then
-  ERT_RUNNER_EMACS="emacs"
-else
-  ERT_RUNNER_EMACS="${EMACS:-emacs}"
-fi
+function inside_emacs_24 {
+  if [[ -n "$INSIDE_EMACS" ]]; then
+    ## Emacs-24 sets INSIDE_EMACS to t with M-x compile, and to a string starting
+    ## with version numbers in shell
+    if [[ "$INSIDE_EMACS" == "t" ]] || [[ "$INSIDE_EMACS" == 24* ]]; then
+      return 0
+    fi
+  fi
+  return 1
```

Comment:
This appears to be equivalent to `[[ -n $INSIDE_EMACS ]] && ! ([[ "$INSIDE_EMACS" == "t" ]] || [[ "$INSIDE_EMACS" == 24* ]])`.  Please simplify.